apps/nowcasting-app/components/charts/remix-line.tsx

```diff
@@ -483,12 +483,16 @@ const RemixLine: React.FC<RemixLineProps> = ({
                         typeof value[1] !== "number"
                       )
                         return null;
-                      if (key === "PROBABILISTIC_RANGE" && (value[0] < 0 || value[1] < 0)) return 0;
+                      if (
+                        key === "PROBABILISTIC_RANGE" &&
```

Comment:
hi @peterdudfield, @braddf and I had a look at the code for handling negative values, and this is what we came up with. Unless the value is  `-.1`,  the value will show as 0.0. Otherwise, neither of the plevel values will be returned. 

This should take care of any negative values showing up at this point. I tested this by putting in values for `value = [x, y]`. Let me know if you're happy for this to be merged into dev.

Reply:
so if the value is `-0.2`, neither will show is that right?, and if its between -0.1 and 0, this will show 0?

Reply:
`value = - 0.05` => shows  `0.1` (just noticed this one)
`value > - 0.05 ` => shows `0.0`
`value < - 0.05` => `neither values` show